Composer/packages/client/src/components/ProjectTree/ProjectTreeHeader.tsx

```diff
@@ -51,54 +118,47 @@ export const ProjectTreeHeader: React.FC<ProjectTreeHeaderProps> = ({ menu }) =>
     };
   });
 
-  const onRenderOverflowButton = (isActive: boolean) => {
-    const moreLabel = formatMessage('Actions');
-    return (overflowItems: IContextualMenuItem[] | undefined) => {
-      if (overflowItems == null) return null;
-      return (
-        <TooltipHost content={moreLabel} directionalHint={DirectionalHint.rightCenter}>
-          <IconButton
-            ariaLabel={moreLabel}
-            className="project-tree-header-more-btn"
-            data-is-focusable={isActive}
-            data-testid="projectTreeHeaderMoreButton"
-            menuIconProps={{ iconName: 'Add', style: { color: NeutralColors.black } }}
-            menuProps={{ items: overflowItems }}
-            role="cell"
-            styles={{ ...moreButton(true), rootHovered: { background: 'none' } }}
-            onKeyDown={(e) => {
-              if (e.key === 'Enter') {
-                e.stopPropagation();
-              }
-            }}
-          />
-        </TooltipHost>
-      );
-    };
+  const handleSearchBoxBlur = (_e) => {
+    onFilter(_e, '');
+    setShowFilter(false);
   };
 
   return (
-    <div css={headerWrapper}>
-      <div css={headerText}>
+    <div css={headerText}>
+      {showFilter ? (
+        <SearchBox
+          underlined
+          ariaLabel={formatMessage('Type trigger name')}
+          componentRef={searchBoxRef}
+          iconProps={{ iconName: 'Filter' }}
+          placeholder={formatMessage('Filter by trigger name')}
+          styles={searchBox}
+          onBlur={handleSearchBoxBlur}
+          onChange={onFilter}
+        />
+      ) : (
         <OverflowSet
```

Comment:
why do you need this overflow set? Simply used a commandbar button
(https://developer.microsoft.com/en-us/fluentui#/controls/web/button)